api/turing/models/log_config.go

```diff
@@ -52,6 +52,10 @@ type KafkaConfig struct {
 	Topic string `json:"topic"`
 	// Serialization Format used for the messages
 	SerializationFormat SerializationFormat `json:"serialization_format"`
+	// Producer Config - Max message byte to send to broker
+	MaxMessageBytes int `json:"max_message_bytes"`
```

Comment:
IMO, these values can just be applied at deploy time, like many other default values we have in `RouterDefaults` (since they are currently applicable to all routers and not user-configured). Any reason we are saving this to the DB?